src/_guides/libraries/private-files.md

```diff
@@ -112,3 +114,16 @@ that make debugging easier.
 For more information on generating and debugging
 JavaScript source for Dart apps, see the
 [dart2js documentation]({{site.webdev}}/tools/dart2js).
+
+### *.js
+
+Calling dart2js without any flags generates `out.js`, but the
+tools that invoke dart2js often specify an output name, such as:
+
+{% prettify none  %}
+dart2js -o main.dart.js main.dart
+{% endprettify %}
+
+Don't commit JavaScript files generated by dart2js. Just be careful
```

Comment:
An idea - what if we combine this and the previous section so that it's all one idea?

For example:
> Don't commit JavaScript files generated by dart2js. Just be careful that you don't accidentally exclude source files written in JavaScript.

> Calling dart2js without any flags generates `out.js`, but the tools that invoke dart2js often specify an output name, such as:

> {% prettify none  %}
dart2js -o main.dart.js main.dart
{% endprettify %}

> For more information on generating and debugging JavaScript source for Dart apps, see the JavaScript source for Dart apps, see the [dart2js documentation]({{site.webdev}}/tools/dart2js). [dart2js documentation]({{site.webdev}}/tools/dart2js).